packages/server/src/bandit/banditSelection.ts

```diff
@@ -57,7 +66,9 @@ export function selectVariantUsingEpsilonGreedy(banditData: BanditData[], test:
     // Choose at random with probability epsilon
     const random = Math.random();
 
-    if (EPSILON > random) {
+    const EPSILON = epsilonValueForBanditTest(testBanditData);
+
+    if (EPSILON > random && EPSILON > 0.5) {
```

Comment:
do we need `EPSILON > 05` here? Seems unnecessary

Reply:
yes, will remove that.